CMSgov/easi-app
# [EASI-4138] FE: Use new "Bookmarked" property on CEDAR System type

# EASI-4138


## Changes and Description

- Explicit refetch queries for bookmarks is replaced with the `refetchQueries` mutation option
- New `GetCedarSystemIsBookmarkedQuery` for the refetching
- Remove the older `GetCedarSystemBookmarksQuery` & `GetCedarSystemsAndBookmarksQuery`



## How to test this change

- Check bookmark toggles in associated System views
  1. Initial queries get the newer `isBookmarked` property
      - for the System list `GetCedarSystems`
      - Profile `GetSystemProfile`
      - Workspace `GetSystemWorkspace` 
  2. Subsequent bookmark toggling will update the value with `Create|DeleteCedarSystemBookmark`
  3. Refreshing the page on views with the initial queries will reflect the `isBookmarked` property update correctly

- QA the Systems list view with bookmark cards and table for bookmarking interaction regressions
